hieradata/staging.yaml

```diff
@@ -23,6 +23,13 @@ govuk::apps::email_alert_api::govdelivery_account_code: 'UKGOVUK'
 govuk::apps::email_alert_api::govdelivery_hostname: 'stage-api.govdelivery.com'
 govuk::apps::email_alert_api::govdelivery_public_hostname: 'stage-public.govdelivery.com'
 govuk::apps::email_alert_api::govuk_notify_template_id: '2844a647-6bf1-4b01-a25c-569d2cc00849'
+govuk::apps::email_alert_api::email_address_override: 'simulate-delivered@notifications.service.gov.uk'
+govuk::apps::email_alert_api::email_address_override_whitelist:
```

Comment:
Might just want to add this in the common one as it'll only take effect if email_address_override is set.

If so could you also include the emails from: https://trello.com/c/wxCFNWrk/609-recruit-and-set-up-our-little-gang-of-testers

Reply:
Yep, good point. Fixed that in 2ac79b7.